README.md

```diff
@@ -1,2 +1,63 @@
-# rfcs
-Repo for managing Requests For Comments (RFCs) for the ESLint project
+# ESLint RFCs
+
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put
```

Comment:
Nit: Would love to see comma before and after "though", but might make the sentence a bit punctuation-noisy